### PR TITLE
improve decollate test for set_determinism

### PR DESCRIPTION
**Pull request by rijobro**
Signed-off-by: Richard Brown <33289025+rijobro@users.noreply.github.com>

### CODE REVIEW

File: tests/test_decollate.py [hunk 1]
```diff
@@ -82,7 +73,7 @@ def test_decollation(self, _, data, batch_size=2, num_workers=2):
             transforms = Compose([LoadImaged("image"), transforms])
 
         dataset = CacheDataset(data, transforms, progress=False)
-        loader = DataLoader(dataset, batch_size=batch_size, shuffle=True, num_workers=num_workers)
+        loader = DataLoader(dataset, batch_size=batch_size, shuffle=False, num_workers=0)
```
Comment (wyli) [inline tests/test_decollate.py:76]: does the previous value cause errors?

**Review by wyli — APPROVED** [review]

File: tests/test_decollate.py (hunk 1 above)
Comment (rijobro) [reply tests/test_decollate.py:76]: Shuffling should cause a problem, since it's compared with images directly from the dataset which aren't shuffled. I'll revert the `num_workers` since that doesn't cause any problems.
